application-forum-ui/src/main/resources/ForumCode/NewFlagSheet.xml

```diff
@@ -84,19 +81,17 @@
 
   $services.localization.render('conversations.flag.success.dialog.message')
 
-  #if ("$!{adminEmail}" != '')
-    #set ($mailTo = "${adminEmail}")
-    #if ("$!{moderatorEmail}" != '')
-      #set ($mailTo = "${mailTo}, {moderatorEmail}")
+  #if ("$!adminEmail" != '')
+    #set ($mailTo = $adminEmail)
+    #if ("$!moderatorEmail" != '')
+      #set ($mailTo = "$mailTo, $moderatorEmail")
```

Comment:
It looks like the moderator email will be set only if the admin email exists. I think there shouldn't be any dependency between them and the email should be sent to any of them. WDYT?

Reply:
You're right. I'll change that.